crates/libs/bindgen/src/rust/extensions/impl/Foundation/Collections/Iterable.rs

```diff
@@ -7,17 +7,18 @@ where
     values: Vec<T::Default>,
 }
 
-impl<T> IIterable_Impl<T> for StockIterable<T>
+impl<T> IIterable_Impl<T> for StockIterable_Impl<T>
 where
     T: windows_core::RuntimeType,
     T::Default: Clone,
 {
     fn First(&self) -> windows_core::Result<IIterator<T>> {
-        unsafe {
-            // TODO: ideally we can do an AddRef rather than a QI here (via cast)...
-            // and then we can get rid of the unsafe as well.
-            Ok(StockIterator { owner: self.cast()?, current: 0.into() }.into())
-        }
+        use windows_core::IUnknownImpl;
+        Ok(windows_core::ComObject::new(StockIterator {
+            owner: self.to_object(),
+            current: 0.into(),
+        })
+        .into_interface())
```

Comment:
I wonder, can we implement `Into`/`From` here as a shorthand for the simple case of returning the interface as before - then it would be similar to before: `Ok(StockIterator {...}.into())` ?

Reply:
The `into()` thing still totally works.  I just like the `ComObject::new(...).into_interface()` more because it's more explicit.